src/scss/navigation/sidenav/_sidenav-primary.scss

```diff
@@ -0,0 +1,45 @@
+// 
+// Sidenav Primary
+// 
+
+// Primary color theme for sidenav
+// Append .sb-sidenav-primary to .sb-sidenav to use
+
+.sb-sidenav-primary {
+    background-color: $sidenav-primary-bg;
```

Comment:
perhaps if we add below or in place of `background-color`:
```css
background-image: linear-gradient(180deg, #4e73df 10%, #224abe 100%);
```
that would bring back that slightly paler blue from the sb-admin-2 theme